### ISSUE TITLE
Support for clean up profile?

### ISSUE THREAD

**Issue by mauricio29** [revelc/formatter-maven-plugin#293]
I would like to correct fewer things, like indentation and remotion of unused imports only.
But formatter does more modifications that can't be disabled.

The cleanup option is perfect for minor corrections. Would you add this goal? 
It could be executed with **> mvn formatter:cleanup** .

In Eclipse cleanup runs with both selected formatter profile and cleanup options which can be customized too.

Examples of cleanup customizations:

![cleanup - builtin profile](https://user-images.githubusercontent.com/10423402/50847559-ae0e6080-1350-11e9-9899-53a868040380.png)

![cleanup - custom profile](https://user-images.githubusercontent.com/10423402/50847564-afd82400-1350-11e9-9a10-6d1e8307079d.png)

**Comment by ctubbsii** [comment]
I think my comment at https://github.com/revelc/formatter-maven-plugin/issues/97#issuecomment-435126457 applies here as well. It'd be better if the additional options available in this Eclipse feature were added to their formatter... where a user can make use of it.

I don't know if the clean up tasks have an easily accessible API in the JDT code like the formatter does, for us to use it. If you want to create a goal and do a pull request, it's fine with me, but I think it's better if upstream merges the functionality into their existing formatter, rather than those actions being only available in a separate API.

As for the imports, like @hazendaz said, I created https://github.com/revelc/impsort-maven-plugin for managing imports, and through some helpful contributions, we were able to implement the removal of unused imports nicely.
